src/components/pages/StudentsPage/WelcomeMessage.tsx

```diff
@@ -0,0 +1,29 @@
+/** @jsxImportSource @emotion/react */
+
+import { Box, Typography, Link } from '@mui/material';
+
+import unpairedMessageCSS from './WelcomeMessage.css';
```

Comment:
Renaming this variable name as per previous comments
```suggestion
import welcomeMessageCSS from './WelcomeMessage.css';
```